aqt/updater.py

```diff
@@ -102,6 +102,11 @@ def _detect_qmake(self) -> bool:
                 return True
         return False
 
+    def patch_prl(self, oldvalue):
+        for prlfile in self.prefix.joinpath("lib").glob("*.prl"):
+            self.logger.info("Patching {}".format(prlfile))
+            self._patch_textfile(prlfile, oldvalue, "$$[QT_INSTALL_LIBS]")
```

Comment:
I am curious about this value: `$$[QT_INSTALL_LIBS]`

I've never seen it before, but it looks like it's internal to qmake: https://doc.qt.io/qt-6/qmake-environment-reference.html

Should we be using these built-in variables to patch other values as well? It seems inconsistent to hard-code specific paths in some places, and qmake variables in others. Personally, I like the idea of using qmake variables in more places, as long as it makes sense.

Reply:
I treat `built-in` and `internal use only` terms differently, and propose to use built-ins instead of hard-coded paths. `qmake -query` shows built-ins, and for cross-platform environment these vars are usually well-defined. Worth noting, that the decision between `QT_INSTALL_LIBS`  vs. `QT_HOST_LIBS` (and similar variables) should be done each time. 
I can see these vars are actively used in installed prf and prl files. 
Also, there are `/raw`, `/dev` , and sometimes `/get` properties, which are occasionaly useful. One can find examples in /mkspecs, f.e. `$$[QT_INSTALL_LIBS/raw]`